package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "react-repo-card",
-  "version": "1.2.1",
+  "version": "1.2.2",
```

Seems more like a minor version bump to me

```suggestion
  "version": "1.3.0",
```